updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding openshift upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} 4 upgrade duration varies based on the deployment topology. This section describes the approximate time duration of each component that follows through during the cluster update. It also explains the factors that affect the upgrade duration and the applicable {product-title} parameters.
```

Comment:
nit: OpenShift 4 has been out for a while now, and folks should be fairly clear that they're looking at v4 docs when they read this.  I dunno if we need to include the 4 here.  But :shrug:, I'm not strongly against including it if other folks think it's useful.

Reply:
@wking I agree with you. I will remove "4".